### ISSUE TITLE
Error: No available storage method found when devtools is open

### ISSUE THREAD

**Issue by fan17** [localForage/localForage#1053]
I am using IndexedDb driver, browser chrome.
If devtools are close on page reload everything is working, but is they are open I see a blank page and an error in console `Error: No available storage method found `.

Here is example code: https://codesandbox.io/s/localforage-devtools-pg5zs

To get error go to the link with opened devtools

**Comment by lincolnthree** [comment]
I can also reproduce this.

**Comment by adebear** [comment]
II am also having this same issue

**Comment by derHodrig** [comment]
Same issue with Angular 14, but only in Firefox. Blink browsers like chrome are okay, Edge works also fine.

**Comment by gplz** [comment]
I had the same issue when I was simulating a device with devtools open. Selecting "Responsive" did not give me the error.

<img width="315" alt="image" src="https://user-images.githubusercontent.com/844688/197235969-87e64eaf-eaaa-433d-ba08-38c4fe36e628.png">

**Comment by lincolnthree** [comment]
Yep, the fix for this issue is stuck in limbo here: https://github.com/localForage/localForage/pull/999

**Comment by Antsthebul** [comment]
Seems like this is a Chrome issue. Localforage works in Edge browser just fine

**Comment by lincolnthree** [comment]
> Seems like this is a Chrome issue. Localforage works in Edge browser just fine

It does appear to be Chrome specific, yes.
